planning/behavior_path_planner/autoware_behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -628,10 +628,10 @@ BehaviorModuleOutput StartPlannerModule::plan()
   if (!status_.found_pull_out_path) {
     RCLCPP_WARN_THROTTLE(
       getLogger(), *clock_, 5000, "Not found safe pull out path, publish stop path");
-    const auto output = generateStopOutput();
+    const auto stop_output = generateStopOutput();
```

Comment:
@kobayu858 
Please move the declaration `BehaviorModuleOutput output;` to line 676.
The same applies to `planWaitingApproval` function.

Reply:
Thanks for the advice. I have moved it as you suggested to narrow the scope range.